vite.config.js

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from "vite";
```

Comment:
@mshrtsr 

I see if we rename it to `vite.config.ts`, we have to put this file onto the `src` directory.
But, this is a configuration file, so I think we should put it in the root directory.
Maybe, we can rename it after we migrate to esm.
What do you think?

Reply:
@hung-cybo 

Updating tsconfig.json as follows.
- change rootDir to `./` (repostitory root)
- add `includes` and `excludes`

```json
{
  "compilerOptions": {
     ...
     "rootDir": "./",
     ...
  },
  "include": ["src/**/*", "vite.config.ts"],
}
```

However, the `rollup-plugin-multi-input` shows an error on TypeScript because there is no type definition of it.
So at this time, keep `vite.config.js` as it is.

Reply:
@mshrtsr 

Thank you.

> However, the rollup-plugin-multi-input shows an error on TypeScript because there is no type definition of it.
Yes, I also faced this problem this morning when trying to move to `.ts`.

But, in case, we must use `vite.config.ts`, we need to define the type of `rollup-plugin-multi-input` by ourselves, isn't it?